docs_2.0/manual_EN/3.quick-start/4.nebula-graph-crud.md

```diff
@@ -488,7 +489,7 @@ You can add indexes to tags or edge types with the `CREATE INDEX` statement. For
 
 ### Must-read for using index
 
-* Appropriate use of indexes can speed up queries, but indexes also decrease the write performance. **DO NOT** use indexes in production environments unless you are fully aware of their influences on your service.
+* Applicable use of indexes can speed up queries, but indexes also decrease the write performance. **DO NOT** use indexes in production environments unless you are fully aware of their influences on your service.
```

Comment:
"Appropriate" means "suitable, acceptable or correct for the particular circumstances". Describes something that is just right.
"Applicable" means that something can be used, nothing more.
Since the use of indexes can be quite tricky, maybe "appropriate" is the more appropriate word here while "applicable" may not be so applicable.

Reply:
I don’t agree. Appropriate is not allowed in the STE standard. I want to use an easy world here.
What about correct?

Reply:
Correct is better 👍